README.md

```diff
@@ -0,0 +1,178 @@
+# slsa-github-generator
+
+This repository contains a reference implementation for generating non-forgeable
+[SLSA provenance](https://slsa.dev/) that meets the requirements for the [SLSA
+level 2 and above](https://slsa.dev/spec/v0.1/levels) using a [Github Actions
+reusable
+workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows).
+
```

Comment:
Shall we cover the API or wait till we've used it with the Go reusable workflow?

Reply:
I was going to give it a bit more time and add something in a follow up PR later. Right now just want to focus on the reusable workflow.